nexclipper-agent/1.10/README.md

```diff
@@ -0,0 +1,40 @@
+# How to use NexClipper-Agent with DC/OS
+
+[NexClipper][nexclipper] is the container monitoring and performance management service specialized in Docker, Apache Mesos, Marathon, DC/OS, Mesosphere, Kubernetes. In this beta version, it support fundamental function to monitor and analyze container cluster(Mesos, DC/OS, Mesosphere, Kubernetes), Container, Agent, event management, resource tracing, performance monitor and basic root cause functionality for event management with SaaS version. It provide more rich functionality for root cause (event management, resource tracing, performance monitor and transaction tasing, etc) soon.
```

Comment:
is a container monitoring and performance management service for Docker, Apache Mesos, Marathon, DC/OS, Mesosphere, Kubernetes. In this beta version, it supports monitoring and analysis of the container cluster (Mesos, DC/OS, Mesosphere, Kubernetes), Container, Agent, event management, resource tracing, performance monitor and basic root cause functionality for event management with SaaS version. Future versions will provide rich functionality for root cause (event management, resource tracing, performance monitor and transaction tasing, etc) soon.